src/torusdk/compat/key.py

```diff
@@ -64,7 +64,7 @@ def classic_key_path(name: str) -> str:
     """
 
     home = Path.home()
-    root_path = home / ".commune" / "key"
```

Comment:
This is wrong. The `compat` package is for retro compatibility with commune formats. It's not to load "Torus native" formats.

Tha's on torusdk/key.py